tests/jepsen.clickhouse-keeper/src/jepsen/clickhouse_keeper/main.clj

```diff
@@ -1,3 +1,98 @@
+(ns jepsen.control.sshj
+  (:require [jepsen.control [core :as core]
+                            [sshj :as sshj]]
+            [slingshot.slingshot :refer [try+ throw+]])
+  (:import (net.schmizz.sshj SSHClient
+                            DefaultConfig)
+           (net.schmizz.sshj.transport.verification PromiscuousVerifier)
+           (java.util.concurrent Semaphore)))
+
+(defrecord SSHJRemote [concurrency-limit
+                       conn-spec
+                       ^SSHClient client
+                       ^Semaphore semaphore]
+  core/Remote
+  (connect [this conn-spec]
+    (if (:dummy conn-spec)
+      (assoc this :conn-spec conn-spec)
+      (try+ (let [c (as-> (SSHClient.) client
+                      (do
+                        (if (:strict-host-key-checking conn-spec)
+                          (.loadKnownHosts client)
+                          (.addHostKeyVerifier client (PromiscuousVerifier.)))
+                        (.connect client (:host conn-spec) (:port conn-spec))
+                        (auth! client conn-spec)
+                        client))]
```

Comment:
https://github.com/jepsen-io/jepsen/pull/554 contains just the diff

Reply:
So most of the code in this PR is copied from upstream and was written by someone else?

(btw., quite interesting to see Clojure code in the wild)

Reply:
yes, the interesting thing about clojure is how you just define the method/record in the same namespace and the modified version will be used.
so you can use a package and overwrite just a single function without a need to do forks/patches etc.